NineChronicles.RPC.Shared/Services/IBlockChainService.cs

```diff
@@ -12,6 +12,8 @@ public interface IBlockChainService : IService<IBlockChainService>
         UnaryResult<byte[]> GetState(byte[] addressBytes);
 
         UnaryResult<byte[]> GetBalance(byte[] addressBytes, byte[] currencyBytes);
+        
+        UnaryResult<byte[]> GetTip(byte[] addressBytes);
```

Comment:
Why does it need `addressBytes` to get tip? Could you describe its purpose? If `tip` means `Libplanet.Blockchain.BlockChain<T>.Tip`, it may not need `Address` 🤔

Reply:
> Why does it need `addressBytes` to get tip? Could you describe its purpose? If `tip` means `Libplanet.Blockchain.BlockChain<T>.Tip`, it may not need `Address` 🤔

Oh yes. It's correct that the function doesn't need any arguments